tests/client_fixture.py

```diff
@@ -51,6 +52,18 @@ def profile_test_client():
     Base.metadata.drop_all(bind=test_engine)
 
 
+@pytest.fixture(scope="session")
+def home_test_client():
```

Comment:
It's a bit pity that `agenda_test_client`,  `profile_test_client` and this are very similar.
I wonder if there is a pritter way for that?